.teamcity/settings.kts

```diff
@@ -36,6 +36,7 @@ version = "2024.12"
 project {
     params {
         java8Home(Os.linux)
+        java17Home(Os.linux)
```

Comment:
🤔 I don't understand why you need to do this. The OS configuration should take care of it already: https://github.com/gradle/dv/blob/4dcb421c32bc668e2891d429ecf2501ae918de17/.teamcity/src/main/kotlin/com/gradle/teamcity/dsl/DslExtensions.kt#L967-L984